packages/inter-protocol/src/auction/auctionBook.js

```diff
@@ -137,17 +134,17 @@ export const prepareAuctionBook = (baggage, zcf, makeRecorderKit) => {
       // may be offers in the book, but these seats will be empty, with all assets
       // returned to the funders.
       const { zcfSeat: collateralSeat } = zcf.makeEmptySeatKit();
-      const { zcfSeat: currencySeat } = zcf.makeEmptySeatKit();
+      const { zcfSeat: bidSeat } = zcf.makeEmptySeatKit();
```

Comment:
I don't like this one, since there are also seats representing users' bids. What do you think of `bidHolderSeat`? The distinction is that it's not involved in the bidding side of the transaction, it's holding amounts in the Bid token.  The best term I can come up with is "offer", but that's overused in Zoe. How about `proffer`?